spec/requests/error_request_spec.rb

```diff
@@ -23,23 +23,20 @@
         expect(parsed_response[:errors][0][:status]).to be_a(String)
 
         expect(parsed_response[:errors][0][:source]).to be_a(Hash)
-        expect(parsed_response[:errors][0][:source].keys).to eq([:pointer, :parameter])
+        expect(parsed_response[:errors][0][:source].keys).to eq([:pointer])
 
         expect(parsed_response[:errors][0][:source][:pointer]).to be_a(String)
-        expect(parsed_response[:errors][0][:source][:parameter]).to be_a(String)
       end
 
       it "response includes detailed informaiton about the error" do 
-        get "/api/v1/usertypedinwhatevertheywanted-notanendpoint!"
+        get "/api/v11/gameplay"
```

Comment:
is there a specific reason you chose to change the get request to a version that doesn't exist vs a non-existing endpoint?

Reply:
Yes, to test a variation of the wrong endpoint being set.